pkg/kubelet/kubelet.go

```diff
@@ -1077,6 +1077,12 @@ func (kl *Kubelet) syncPod(pod *api.BoundPod, dockerContainers dockertools.Docke
 					containersToKeep[containerID] = empty{}
 					continue
 				}
+				ref, ok := kl.getRef(containerID)
+				if !ok {
+					glog.Warningf("No ref for pod '%v' - '%v'", ID, name)
+				} else {
+					record.Eventf(ref, "unhealthy", "Health Check Failed %v - %v", ID, name)
```

Comment:
Please don't use the term "Health Check". We now have both liveness and readiness probes in the API. You're reporting that the liveness probe failed.